apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/shareEmbedModal/personal-links-tab.test.tsx

```
@@ -0,0 +1,519 @@
+  DatePicker: ({ date, updateSurveyDate, minDate, onClearDate }: any) => (
+    <div data-testid="date-picker">
+      <input
+        data-testid="date-input"
+        type="date"
+        value={date ? date.toISOString().split("T")[0] : ""}
+        onChange={(e) => {
+          const newDate = e.target.value ? new Date(e.target.value) : null;
+          updateSurveyDate(newDate);
+        }}
+      />
+      <button data-testid="clear-date" onClick={() => onClearDate()}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)